src/s1_orbits/s1_orbits.py

```diff
@@ -0,0 +1,42 @@
+import logging
+from pathlib import Path
+from typing import Union
+
+import requests
+
+
+API_URL = "https://s1-orbits.asf.alaska.edu/scene"
+
+
+def fetch_for_scene(
+    scene: str,
+    dir: Union[Path, str] = ".",
+) -> str:
+    """For the given scene, this downloads the AUX_POEORB file, if available, otherwise it downloads the AUX_RESORB.
+
+    Args:
+        scene: The scene name for which to download the orbit file.
+        dir: The directory that the orbit file should download into.        
+
+    Returns:
+        download_path: The path to the downloaded file.
+    """
+    request_url = f"{API_URL}/{scene}"
+
+    logging.info(f"Downloading {request_url}")
+
+    session = requests.Session()
+
+    with session.get(request_url, stream=True) as res:
```

Comment:
RESORB files are 590 KB and POEORB files are 4.4 MB, I don't think there's a need for streaming here.

I think saving to disk becomes `download_path.write_text(res.text)`